app/controllers/admins/service_admins_controller.rb

```diff
@@ -96,7 +96,8 @@ def record_number_options
 
   def search_service_admins
     if params[:q].present?
-      @service_admins = @service_admins.where(name: params[:q][:name] ) unless params[:q][:name].blank?
+      @service_admins = @service_admins
+                            .where(id: params[:q][:id]) unless params[:q][:id].blank?
```

Comment:
Line is too long. [85/80]<br>Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.